src/tests/TestServerResolver/TestServerResolver.cpp

```diff
@@ -36,6 +37,11 @@ void TestServerResolver::simpleSrv() {
 	return;
 #endif
 
+	// Qt 5's SRV resolver does not work in Wine.
```

Comment:
Can we provide more information here? Maybe https://github.com/mumble-voip/mumble/pull/3277#issuecomment-354687115

> QDnsLookup::ServerFailureError | 5 | the server encountered an internal failure while processing the request (SERVFAIL).

Reply:
I should probably just file the WineHQ bug :-)

Reply:
Done.